packages/tools/kadena-cli/src/utils/globalOptions.ts

```diff
@@ -520,7 +520,7 @@ export const globalOptions = {
     prompt: security.securityPasswordVerifyPrompt,
     validation: z.string(),
     option: new Option(
-      '-p, --security-verify-password <securityVerifyPassword>',
+      '-vp, --security-verify-password <securityVerifyPassword>',
```

Comment:
we need to remove all multi-character options with a single `-`, in this case just remove `-vp` entirely I think.